### DISCUSSION TITLE
Softer background color in alert?

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#2760] [Q&A] [answered]
How can I make the background color for alert-info, alert-success, etc., softer?

Maybe changing the opacity would work, or should I customize the colors?

https://daisyui.com/components/alert/

**Comment by saadeghi** [comment, marked as answer]
You can set a custom color for `success`, `info`, etc. in your theme:  
https://play.tailwindcss.com/6l3CrrduRa?file=config

Let me know if you have a question.

**Reply by jaimeiniesta** [reply]
Awesome! Thanks, it works great.
